Makefile

```diff
@@ -73,11 +85,15 @@ test:
 
 .PHONY: test-386
 test-386:
-	set -e; for dir in $(ALL_GO_MOD_DIRS); do \
-	  echo "go test ./... GOARCH 386 in $${dir}"; \
-	  (cd "$${dir}" && \
-	    GOARCH=386 $(GOTEST_MIN) ./...); \
-	done
+	if [ $(SKIP_386_TEST) = true ] ; then \
+	  echo "skipping the test for GOARCH 386 as it is not supported on the current OS"; \
+	else \
+      set -e; for dir in $(ALL_GO_MOD_DIRS); do \
+        echo "go test ./... GOARCH 386 in $${dir}"; \
+        (cd "$${dir}" && \
+          GOARCH=386 $(GOTEST_MIN) ./...); \
+	  done; \
```

Comment:
still not indented properly. I'll remove the makefile change and create a separate PR.

Reply:
Alright, thanks.